src/ProjectTemplates/Web.Spa.ProjectTemplates/content/Angular-CSharp/ClientApp/src/api-authorization/authorize.service.ts

```diff
@@ -0,0 +1,324 @@
+import { Injectable } from '@angular/core';
```

Comment:
There are dozens of TSLint errors from this file. Please open it in VS Code to see them :)

Reply:
I didn't realize the TSLint errors as building was fine (my testing scripts do ng build)

Don't bother about the linting errors, I'll open vs code and clean all of them.

Reply:
Also, again the `oidc-client` flows are outside my knowledge, so I'm not commenting on the correctness or completeness of these. Again, looks totally plausible though!

Reply:
Fixed all the TSLint errors